api/routes/addEntries.js

```diff
@@ -1,37 +1,27 @@
+//Add New entries
 const express = require('express')
 const route = express.Router()
 const database = require('../models/post.model')
-
 route.post('/addEntries', (req,res)=>{
-    const {title, description} = req.body
-    const databaseLength = database.Posts.length
-    let oldId = database.Posts[databaseLength - 1].id
-    let newId = oldId 
-    database.Posts.push({
-        id:newId + 1,
-        title:title,
-        description:description,
-        createdOn: Date()
-    })
+const {title, description} = req.body
+const databaseLength = database.Posts.length
+let oldId = database.Posts[databaseLength - 1].id
+let newId = oldId 
+database.Posts.push({
+id:newId + 1,
+title:title,
```

Comment:
What if the users sends no title and no description? What if they send only spaces as title and description?
How will you handle that? Implement it here.

Reply:
You can use `@joi`,  it is a powerful schema description language and data validator for JavaScript `npm i @joi`